src/module.c

```diff
@@ -402,7 +402,7 @@ int TSDB_queryindex(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     return REDISMODULE_OK;
 }
 
-int TSDB_mrange(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+int TSDB_generic_mrange(RedisModuleCtx *ctx, RedisModuleString **argv, int argc, bool rev) {
```

Comment:
`rev` should be options

Reply:
chunk has options
iterator only has bool

Reply:
removed option where bool is sufficient